src/components/BooksForm.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const categories = [
+  'Action',
+  'Biography',
+  'History',
+  'Horror',
+  'Kids',
+  'Learning',
+  'Sci-Fi',
+];
+
+function BooksForm() {
+  return (
+    <form>
+      <label htmlFor="title">
+        Book Title:
+        <input type="text" name="title" id="title" required />
+      </label>
+      <br />
+      <select required>
+        {categories.map(category => (
+          <option key={category}>{category}</option>
+        ))}
+      </select>
+      <button type="submit">Submit</button>
+    </form>
+  );
+}
+
+export default BooksForm;
```

Comment:
- place this BOOKSFORM component inside `"
`src/containers`"` folder 
- it should  have a state with `title` and `category`

Reply:
Dont mind putting it in src/containers but can you please help me understand why it should have a state with title and category?

Reply:
Hello @sumaerjolly , yeah you are right........ignore the this second comment, my mistake 😊